chatkit-core/src/main/kotlin/com/pusher/chatkit/state/Actions.kt

```diff
@@ -0,0 +1,27 @@
+package com.pusher.chatkit.state
+
+import com.pusher.chatkit.rooms.state.JoinedRoomInternalType
+
+internal sealed class Action
+
+internal data class JoinedRoomsReceived(
+    val rooms: List<JoinedRoomInternalType>,
+    val unreadCounts: Map<String, Int>
+) : Action()
+
+internal data class JoinedRoom(
+    val room: JoinedRoomInternalType,
+    val unreadCount: Int
+) : Action()
+
+internal data class LeftRoom(
+    val roomId: String
+) : Action()
+
+internal data class DeletedRoom(
```

Comment:
it seems this [bit of feedback](https://github.com/pusher/chatkit-android/pull/192#discussion_r386978334) got missed out

Reply:
Yes :( I noticed that when I tried to merge this branch into #193 - it was just the Actions that were not updated - have since fixed!